pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook.go

```diff
@@ -175,3 +182,21 @@ func (v *fleetResourceValidator) InjectDecoder(d *admission.Decoder) error {
 	v.decoder = d
 	return nil
 }
+
+// isFleetMemberNamespace returns true if namespace is a fleet member cluster namespace.
+func isFleetMemberNamespace(namespace string) bool {
+	return strings.HasPrefix(namespace, "fleet-member")
```

Comment:
Hi Arvind! Instead of hard-coding, I assume this could use the namespaceNameFormat variable as well?